### DISCUSSION TITLE
Only applies theme (and not items) to other applications

### DISCUSSION THREAD

**Discussion by ThePython10110** [moudey/Shell#80] [General]
Shell currently does not apply any of the static/dynamic customizations to other applications' context menus. It only applies the theme. I tested this with Tablacus Explorer and Explorer++.

Windows Explorer works as expected (with my custom menus and a couple of moved items)
![Explorer example](https://user-images.githubusercontent.com/87204246/205536012-ce826d61-1e31-4eaf-bac8-6c152b028ae8.png)
Tablacus doesn't have that; just the theme (although my screenshot makes it hard to tell). "Open in new tab" and "Open in background" are part of Tablacus.
![Tablacus example](https://user-images.githubusercontent.com/87204246/205536013-d30a1a3e-aa29-4aa2-978d-5d69137bab6b.png)

**Comment by moudey** [comment]
Hi @ThePython10110 Please test this [build](https://github.com/moudey/Shell/files/10158264/shell-test.zip)

Set `*` in the `type` property to show the item in all menus

```
dynamic
{
    item(type="*" title='hello from any menu' image=#f00)
    item(type="*" where=window.is_start title='hello from Winx menu' image=#00f)
    item(title='hello from normal menu' image=#0f0)
}
```

**Comment by ThePython10110** [comment]
Now it applies dynamic customization, but nothing static.

**Comment by moudey** [comment]
```
static
{
    item(type='taskbar' find='search' vis=disabled)
    // or
    item(type='*' where=window.is_taskbar find='search' vis=disabled)

    item(type='*' where=window.is_start find='search' vis=disabled)
}
```

**Comment by ThePython10110** [comment]
Some things do work, but changing the parent apparently doesn't. I'm trying to make Git Bash appear under "Terminal," as well as hiding disabled items. It works great in Explorer, but not in Tablacus. 

`item(type='*' find='Git Bash here|Powershell' parent='Terminal')`

```
menu(type='taskbar' title='Terminall' image=\uE0D6)
		{
			item(title='CMD' icon='cmd.exe' admin=@key(key.shift) cmd='cmd.exe')
		}
```

Explorer:
![image](https://user-images.githubusercontent.com/87204246/205772103-18b457f8-6e3b-4152-a992-cf50fd2c56ce.png)
Tablacus:
![image](https://user-images.githubusercontent.com/87204246/205772230-0eb49ee1-5738-4436-b4be-7c8a9de0f75d.png)

**Comment by moudey** [comment]
```
static
{
    item(type='*' find='Git Bash here|Powershell' parent='Terminal')
}

dynamic
{
    menu(type='*' title='Terminal' image=\uE0D6)
    {
        item(title='CMD' admin=key.shift() image cmd='cmd.exe')
    }
}
```

**Comment by moudey** [comment]
To remove all disabled items
```
static
{
    item(type='*' where=this.disabled vis=remove)
}
```
